Rakefile

```
@@ -235,8 +235,8 @@ end
 
 def push_to_rubygems(version)
   print "Checking to see if pkg/fig-#{version}.gem exists... "
-  if File.exists?("pkg/fig-#{version}.gem")
-    puts 'File exists.'
+  if File.exist?("pkg/fig-#{version}.gem")
+    puts 'File.exist.'
```

This looks like a regexp replace that got a little too greedy.
```suggestion
    puts 'File exists.'
```

---

Good catch!  Fixed.